src/game/SpellAuras.cpp

```diff
@@ -5724,6 +5724,19 @@ void Aura::PeriodicTick()
                         }
                         break;
                     }
+                    case 46771: {                               // Flame Sear
+                        if (target->GetTypeId() == TYPEID_PLAYER) {
+                            if (!target->HasAura(45343) &&
+                                !target->HasAura(47300)) {          // No Dark Flame applied. There are two auras
+                                if (m_periodicTick % 4 == 1) {      // Only doze the aura every 2 ticks (should maybe be one second cooldown)
+                                    target->CastSpell(target, 45348,
```

Comment:
I believe you should apply different debuffs on 45343 presence, and different on 47300. Right now you apply only fire one.